components/validation/form-element-mixin.js

```diff
@@ -0,0 +1,35 @@
+export const FormElementMixin = superclass => class extends superclass {
+
+	constructor() {
+		super();
+		this.__hiddenInput = document.createElement('input');
```

Comment:
Is there a reason not to defer this work until `connectedCallback`?

Reply:
Yeah, it'd be fine to defer it. Just threw it there to get the proof of concept functional as fast as possible. Its still pretty rough around the edges.